app/components/root.js

```diff
@@ -44,8 +44,7 @@ class Root extends React.Component {
             <Route exact path="/products/add" component={AddProductForm} />
             <Route path="/products/:id" component={ProductPage} />
             <Route
-              path="/orders/:orderId/shoppingcart/:userId?" /*component={ShoppingCart} */
-            />
+              path="/shoppingcart/:userId?" component={ShoppingCart} />
```

Comment:
does this render?

Reply:
nope. I'll keep the original path and comment out the component.